releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -1537,7 +1537,7 @@
   "124063": {
     "commit": "a7ece470e56f0479573db2632031d274867d96b1",
     "text": "Fixed a bug where hard evictions due to resource pressure allowed pods\nto use the full termination grace period instead of shutting down instantly. \nThis bug also affected force deleted pods. Both cases now receive a termination grace period of 1 second.",
-    "markdown": "Fixed a bug where hard evictions due to resource pressure allowed pods\n  to use the full termination grace period instead of shutting down instantly. \n  This bug also affected force deleted pods. Both cases now receive a termination grace period of 1 second. ([#124063](https://github.com/kubernetes/kubernetes/pull/124063), [@olyazavr](https://github.com/olyazavr)) [SIG Node]",
+    "markdown": "Fixed a bug where hard evictions due to resource pressure allowed pods\n  to use the full termination grace period instead of shutting down instantly. \n  This bug also affected force deleted pods. Both cases now receive a termination grace period of 1 second. ([#124063](https://github.com/kubernetes/kubernetes/pull/124063), [@olyazavr](https://github.com/olyazavr))",
```

Comment:
```suggestion
    "markdown": "Fixed a bug where hard evictions due to resource pressure allowed pods\n  to use the full termination grace period instead of shutting down instantly. \n  This bug also affected force deleted pods. Both cases now receive a termination grace period of 1 second. ([#124063](https://github.com/kubernetes/kubernetes/pull/124063), [@olyazavr](https://github.com/olyazavr)) [SIG Node]",
```